src/WebJobs.Script.WebHost/ContainerManagement/LinuxContainerActivityPublisher.cs

```diff
@@ -92,7 +92,7 @@ public Task StartAsync(CancellationToken cancellationToken)
 
         public Task StopAsync(CancellationToken cancellationToken)
         {
-            _logger.LogInformation($"Stopping {nameof(LinuxContainerActivityPublisher)}");
+            _logger.LogInformation("Stopping {nameof(LinuxContainerActivityPublisher)}", nameof(LinuxContainerActivityPublisher));
```

Comment:
This one should be reverted - it is a constant interpolation, no need to use a structured log